### ISSUE TITLE
how to send web push notification with this package

### ISSUE THREAD

**Issue by mr-kiani** [brozot/Laravel-FCM#151]

**Comment by owaiskiani65** [comment]
Same Question from my side. Can we use this package for web push notification?

**Comment by zenichanin** [comment]
I want to know this too...

**Comment by HugoLuna5** [comment]
That's right, you just have to get the user's token and save it in your DB, in the readme file there is an example of how to send the notification

**Comment by HugoLuna5** [comment]
> La misma pregunta de mi parte. ¿Podemos usar este paquete para notificaciones push web?

That's right, you just have to get the user's token and save it in your DB, in the readme file there is an example of how to send the notification
